Makefile

```diff
@@ -5,7 +5,7 @@ flake:
 	pipenv run flake8
 
 check:
-	PIPENV_PYUP_API_KEY="" pipenv check -i 39611 -i 39608 -i 40014
+	PIPENV_PYUP_API_KEY="" pipenv check -i 39611 -i 39608 -i 40014 -i 40291
```

Comment:
This seems to be a warning about pip versions, it should not be ignored. If you're getting this warning locally try `pip install -U pip` In case it's picking up your local pip version, and `pipenv run pip install -U pip` or deleting and recreating the pipenv to get the pipenv's version up to date. The warning should not be an issue in any of the automated builds because the pipenv is installed fresh each time, it only occurs when longer preserved pipenvs go stale, as far as I've experienced.